globals/matchmaking_utils.go

```diff
@@ -190,144 +206,135 @@ func FindSessionByMatchmakeSession(searchMatchmakeSession *match_making_types.Ma
 			continue
 		}
 
+		// If the session only allows friends, check if the owner is in the friend list of the PID
+		// TODO - Is this a flag or a constant?
+		if sessionToCheck.GameMatchmakeSession.ParticipationPolicy == 98 {
+			if GetUserFriendPIDsHandler == nil {
+				Logger.Warning("Missing GetUserFriendPIDsHandler!")
+				continue
```

Comment:
Since we are ranging every session, continuing allows public matches to work even if friend matches are not implemented.

This could be useful in the weird edge case where we discover a game uses friend matches after we implement the server. If we didn't do this, any public matches created after the friend match wouldn't show up.

Reply:
:+1: